### PR TITLE
Perform a check of required files

### PR DESCRIPTION
**Pull request by vondele**

### CODE REVIEW

File: run_games.py [hunk 1]
```diff
@@ -241,14 +238,34 @@ def main():
...
+    if stockfish_test is None:
+        stockfish_test = stockfish_base
+
+    if not os.path.exists(stockfish_base):
+       sys.exit("Stockfish base does not exist!")
+
+    if not os.path.exists(stockfish_test):
+       sys.exit("Stockfish test does not exist!")
+
+    if not os.path.exists(args.ordo_exe):
```
Comment (Sopel97) [inline run_games.py]: does this respect the `PATH` variable? In my setups I just put "ordo" and assume it can correctly found by the shell.

Comment (Sopel97) [reply run_games.py]: perhaps do some `ordo --version` or an equivalent?

File: run_games.py [hunk 2]
```diff
@@ -241,14 +238,34 @@ def main():
...
+       sys.exit("ordo does not exist!")
+
+    if not os.path.exists(args.c_chess_exe):
```
Comment (Sopel97) [inline run_games.py]: same here

File: run_games.py (hunk 1 above)
Comment (vondele) [reply run_games.py]: OK, good idea, need to check.

Comment (vondele) [reply run_games.py]: This should be resolved using `shutil.which()`

**Pull request merged**